scrimage-core/src/main/java/com/sksamuel/scrimage/nio/JpegWriter.java

```diff
@@ -14,6 +14,7 @@
 public class JpegWriter implements ImageWriter {
 
    public static JpegWriter NoCompression = new JpegWriter(100, false);
```

Comment:
Sorry for reviewing when already merged, but we were using the `NoCompression` jpeg writer. Now it won't work as I see, as MODE_DISABLED is not supported by JPEGImageWriter. The error is `JPEG compression cannot be disabled`.
So I'd suggest this static writer to be deleted as it can't be used with a deprecation message or something pointing to `CompressionFromMetaData` writer.

Reply:
@gipeshka @sksamuel in fact, the `JpegWriter.NoCompression` mode itself is a bit misleading.
In regards to the original JPEG file format there's no such thing as "no compression" JPEG.

There were some separate solutions created though, that enabled loss-less compression on top of "JPEG" (e.g. see "Lossless JPEG" or "JPEG 2000"), but those extensions may not be supported by the javax.imageio subsystem.

Reply:
Thanks I've taken the suggestions and deprecated this static value.